Configuration/StandardSequences/python/Digi_cff.py

```diff
@@ -42,3 +42,45 @@
     from FastSimulation.Configuration.DigiAliases_cff import loadDigiAliases
     loadDigiAliases(premix = False)
     from FastSimulation.Configuration.DigiAliases_cff import generalTracks,ecalPreshowerDigis,ecalDigis,hcalDigis,muonDTDigis,muonCSCDigis,muonRPCDigis
+
+#phase 2 common mods
+def _modifyDigitizerPhase2Common( theProcess ):
+    theProcess.load("CalibCalorimetry/HcalPlugins/Hcal_Conditions_forGlobalTag_cff")
+    theProcess.es_hardcode.toGet = cms.untracked.vstring(
```

Comment:
The module is an ESProducer and when the specified python file is loaded, an es_prefer is included, as well as an instance of HcalDbProducer. So in its current form, I don't think we could load it for all Eras. Should we make the presence or absence of the es_prefer depend on the Era?

https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/CalibCalorimetry/HcalPlugins/python/Hcal_Conditions_forGlobalTag_cff.py